includes/functions/functions_general_shared.php

```diff
@@ -437,3 +437,9 @@ function zen_draw_products_pull_down_categories_attributes($field_name, $paramet
    return zen_draw_pulldown_categories_having_products_with_attributes($field_name, $parameters, $exclude, $show_full_path, $filter_by_option_name);
 }
 
+function zen_get_orders_status() 
+{
+   trigger_error('Call to deprecated function; please use new names', E_USER_DEPRECATED);
```

Comment:
How about we give a clue as to what the new name is?

Reply:
They only have look at the traceback and go to this function, and they only have to do this once.